tests/transports-governance/test_governance_edge_cases_comprehensive.py

```
@@ -0,0 +1,1433 @@
+        for team_id in team_ids:
+            vk_data = test_data_factory.virtual_key_data(team_id=team_id)
+            response = governance_client.create_virtual_key(**vk_data)
+
+            if response.status_code == 201:
+                vk_id = response.json()["id"]
+                track_created_entity("virtual_keys", vk_id)
+                entity_counts["virtual_keys"] += 1
+
+        vk_creation_time = time.time() - vk_start
+
+        total_time = time.time() - start_time
```

Ruff (0.12.2)

Local variable `total_time` is assigned to but never used

Remove assignment to unused variable `total_time`

(F841)